File: web3rpc/rpc-specs/code-samples/python/governance/vote.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

key = "governance.governancemode"
value = "ballot"

w3 = Web3(Web3.HTTPProvider(host))
governance_response = w3.governance.vote(key, value)

print(governance_response)

File: web3rpc/rpc-specs/code-samples/javascript/governance/vote.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const key = "governance.governancemode";
    const value = "ballot";

    provider.governance.vote(key, value, {}, (err, data, response) => {
        console.log(data);
    });
}
)()